src/nikhilMJun20/FindItemId.java

```diff
@@ -0,0 +1,47 @@
+/*Assignment 33 : 22nd July 2020
+
+Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.
+*/
+
+package nikhilMJun20;
+
+public class FindItemId {
+	void displayItemId(String str, String toBeFound) {
+		System.out.println("\nGiven string:-\n"+str);
+		try {
+			String temp = str.substring(str.indexOf(toBeFound));
+			String itemId = "";
+			if(temp.charAt(toBeFound.length()) == '=') {
+				int index = toBeFound.length()+1;
+				while(Character.isDigit(temp.charAt(index))) {
+					itemId += temp.charAt(index);
```

Comment:
Suggesting an alternative You can also write : 
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

This two line in try block and it will work.